editor/src/components/canvas/canvas-strategies/canvas-strategies.tsx

```diff
@@ -127,13 +128,27 @@ const AncestorCompatibleStrategies: Array<MetaCanvasStrategy> = preventAllOnRoot
   dragToMoveMetaStrategy,
 ])
 
+const metaStrategy =
+  (strategy: CanvasStrategyFactory): MetaCanvasStrategy =>
+  (
+    canvasState: InteractionCanvasState,
+    interactionSession: InteractionSession | null,
+    customStrategyState: CustomStrategyState,
+  ): Array<CanvasStrategy> => {
+    return mapDropNulls(
+      (factory) => factory(canvasState, interactionSession, customStrategyState),
+      [strategy],
+    )
+  }
+
 export const RegisteredCanvasStrategies: Array<MetaCanvasStrategy> = [
   ...AncestorCompatibleStrategies,
   preventOnRootElements(resizeStrategies),
   propertyControlStrategies,
   drawToInsertMetaStrategy,
   dragToInsertMetaStrategy,
   ancestorMetaStrategy(AncestorCompatibleStrategies, 1),
+  metaStrategy(keyboardSetFontSizeStrategy),
```

Comment:
why not make `keyboardSetFontSizeStrategy` a `MetaCanvasStrategy` to begin with?

Reply:
I wanted to keep as similar as possible to the rest of the strategies (e.g. `keyboardAbsoluteResizeStrategy`), I'm not sure it makes sense to turn it into a meta strategy just because it's a top-level one